packages/kit/src/context.ts

```diff
@@ -36,5 +45,10 @@ export function useNuxt (): Nuxt {
  * ```
  */
 export function tryUseNuxt (): Nuxt | null {
-  return nuxtCtx.tryUse()
+  const nuxt = nuxtCtx().tryUse()
+  if (!nuxt) {
+    logger.warn('Using fallback global Nuxt instance. You may be using a @nuxt/kit composable outside of a Nuxt context, this behavior is deprecated and will be removed in v4.')
```

Comment:
I'm not sure how to rephrase this to make it sound a bit gentler 😅. Any suggestions?

cc @danielroe

Reply:
this also isn't going to be remove d in V4, maybe in V5 🤔